ConfigurationSystem/Client/LocalConfiguration.py

```diff
@@ -285,13 +285,17 @@ def __parseCommandLine(self):
 
   def __loadCFGFiles(self):
     """
-    Load ~/.dirac.cfg
-    Load cfg files specified in addCFGFile calls
-    Load cfg files with come from the command line
+    Loads possibly several cfg files, in order:
+    1. ~/.dirac.cfg
+    2. cfg files pointed by DIRACSYSCONFIG env variable
+    3. cfg files specified in addCFGFile calls
+    4. cfg files that come from the command line
     """
     errorsList = []
     if 'DIRACSYSCONFIG' in os.environ:
-      gConfigurationData.loadFile(os.environ['DIRACSYSCONFIG'])
+      diracSysConfigFiles = os.environ['DIRACSYSCONFIG'].replace(' ', '').split(';')
```

Comment:
I think this should be in a reversed order. So that the first path in the DIRACSYSCONFIG is taking preference. This would be more intuitive

Reply:
Done.